x-pack/plugins/security_solution/server/lib/detection_engine/rule_actions/migrations.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  SavedObjectUnsanitizedDoc,
+  SavedObjectSanitizedDoc,
+} from '../../../../../../../src/core/server';
+import { IRuleActionsAttributesSavedObjectAttributes, RuleAlertAction } from './types';
+
+export const ruleActionsSavedObjectMigration = {
```

Comment:
Is this the same code as in the `alerts` files above? Do you think it'd be worth creating a single function to do it in both places or are they different?